tests/Areas/AppConfig/UnitTests/KeyValue/KeyValueDeleteCommandTests.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
```

Comment:
Unused import. I think most of these classes stopped using this when you moved onto adding `_command`, `_context`, and `_parsing` class variables. Can you make sure we don't have more unused statements like these in te new tests please? :)